src/tunes.js

```diff
@@ -5,56 +5,93 @@ import { make } from './helpers';
  */
 export default class Tunes {
   /**
-   * @param {{cssClasses: object, settings: object, onTuneToggled: Function}}
-   *  cssClasses - Css class names
-   *  settings - Available Image tunes
-   *  onTuneToggled - Tune toggling callback
-   */
-  constructor({ cssClasses, settings, onTuneToggled }) {
+     * @param {{cssClasses: object, settings: object, onTuneToggled: Function, title: "string"}}
+     *  cssClasses - CSS class names
+     *  settings - Available image tunes
+     *  onTuneToggled - Tune toggling callback
+     *  title - Title string
+     */
+  constructor({
+    cssClasses, settings, onTuneToggled, title,
+  }) {
     this.cssClasses = cssClasses;
     this.onTuneToggled = onTuneToggled;
     this.settings = settings;
+    this.title = title;
     this.buttons = [];
   }
 
   /**
-   * Makes buttons with tunes: add background, add border, stretch image
-   *
-   * @param {Object} data Tool data
-   * @return {HTMLDivElement}
-   */
+     * Makes buttons with tunes: add background, add border, stretch image
+     *
+     * @param {Object} data Tool data
+     * @return {HTMLDivElement}
+     */
   render(data) {
     const wrapper = make('div');
     this.buttons = [];
 
+    const tuneNames = {
+      withBorder: 'Chegara bilan',
```

Comment:
Please apply internalization since so that we can easily change namings from our app without tweaking this repository.
Pleaes use this approach - https://github.com/editor-js/image/blob/927ec04edae75fb2e9a83add24be38d439dc3a19/src/index.js#L238C7-L238C7

Reply:
bunday qila olamizmi? Biz hozir o'zimiz kiritib chiqdikuu boshqattan

Reply:
this.api.i18n orqali qilsez bo'ladi.
this.api.i18n. - Editor orqali render qivotganda shu api keladi.